### PR TITLE
Run cypress tests

### PR DESCRIPTION
**Pull request by harishzen**

### CODE REVIEW

File: bin/runner.js [hunk 1]
```diff
@@ -89,9 +91,49 @@ function main(){
...
+          console.error(err)
+        })
+
+
+      }
+
     });
 
+
+    //if not testem quit
```
Comment (glynn-zenefits) [inline bin/runner.js:130]: (This isn't actually quitting [the process], maybe "We were asked to run cypress so no need run the rest of the code here that boots up testem"?)

File: bin/runner.js [hunk 2]
```diff
@@ -89,9 +91,49 @@ function main(){
       bridge.start();
       process.env.PORT = graphqlUrl.port;
       process.env.BASEURL = apiUrl.href;
+
+      //if not cypress return early
```
Comment (glynn-zenefits) [inline bin/runner.js:95]: Comment should probably be one line down? (Also we're not actually returning early, it's just in an if block, probably this comment was written earlier when the code looked different.)

File: package.json [hunk 3]
```diff
@@ -13,7 +13,8 @@
     "dependency-checker": "git+https://github.com/zenefits/dependency-checker#v0.0.2",
     "npmlog": "^1.2.1",
     "redis": "^0.12.1",
-    "testem": "git+https://github.com/zenefits/testem#v1.6.0-zenefits.2"
```
Comment (glynn-zenefits) [inline package.json:16]: This line shouldn't be removed?

Comment (harishzen) [reply package.json:16]: Ouch. must have been an inadvertent delete

**Comment by MiguelMadero** [comment]
Why is this needed? Why can't we just run cypress directly without having to go through testem?

**Comment by ashish-psingh** [comment]
@MiguelMadero We are trying to run cypress E2E tests present in z-frontend for yp-backend changes.
Doc can be found at https://docs.google.com/document/d/16acgQAZ9hFcYonKBLORcR01pEnqcC99GYFYzXcBiJ0s/edit.

**Comment by MiguelMadero** [comment]
And why do we need to do it via testem-wrap instead of just using cypress directly?

**Pull request merged**